kube_hunter/conf/logging.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+
+DEFAULT_LEVEL = logging.INFO
+DEFAULT_LEVEL_NAME = logging.getLevelName(DEFAULT_LEVEL)
+LOG_FORMAT = "%(asctime)s %(levelname)s %(name)s %(message)s"
+
+
+def setup_logger(level_name):
+    # Remove any existing handlers
+    # Unnecessary in Python 3.8 since `logging.basicConfig` has `force` parameter
+    for h in logging.getLogger().handlers[:]:
+        h.close()
+        logging.getLogger().removeHandler(h)
+
+    if level_name.upper() == "NONE":
+        logging.disable(logging.CRITICAL)
+    else:
+        log_level = logging._nameToLevel.get(level_name.upper(), None)
+        logging.basicConfig(level=log_level or DEFAULT_LEVEL, format=LOG_FORMAT)
+        if not log_level:
```

Comment:
This part has 2 `if`s checking for the same thing in different ways. I think it clearer and less error prone to move the `or DEFAULT_LEVEL` into the `if not log_level` (if you do that you'd need to rearrange the if)

Reply:
If I do this, I'll need to keep a variable that indicates if the log level parameter was valid (and make an `if` to print a warning message).
I think it is less readable. WDUT?

Edit:
Fixed this by using `getattr` and a type check